src/zui/ZUIEditor/EditorOverlays/index.tsx

```diff
@@ -18,10 +20,20 @@ export type BlockDividerData = {
   y: number;
 };
 
+export type BlockType =
```

Comment:
This feels like it should be exported somewhere else, maybe `ZUIEditor/types.ts`?